amaznot/resources/views/layouts/layout.blade.php

```diff
@@ -1,81 +1,248 @@
+        </button>
 
-    <!--HEADER-->
-    <!--END HEADER-->
+        <div class="collapse navbar-collapse ml-auto navbar-items-container" id="navbarSupportedContent">
+          <ul class="navbar-nav ml-auto navbar-items">
+            <li class="nav-item">
+              <a class="nav-link" href="{{ route('home') }}">Home</a>
+            </li>
+            <li class="nav-item dropdown position-static">
+              <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                Aisles
+              </a>
+              <div class="dropdown-menu w-100" aria-labelledby="navbarDropdown">
+                <div class="nav-categories-container">
+                  <div class="category-item">
+                    <h5 class="category-item-header">
+                      Choose a Category
+                    </h5>
+                    @php
+                    $data = array_map(fn($data) => $data['category'], $headerData['array']);
+                    $data = array_unique($data);
+                    @endphp
+                    @foreach ($data as $cat)
+                    @php
+                    $catURL = "https://".$_SERVER['SERVER_NAME']."/products/".$cat;
+                    @endphp
+                    <a class="category-item-title" href="<?php echo $catURL ?>">
+                      {{$cat}}
+                    </a>
+                    @endforeach
+                  </div>
+                  <div class="category-links">
+                    <h5 class="category-item-header">
+                      Choose a Subcategory
+                    </h5>
+                    @php
+                    $subcat = array_filter($headerData['array'], fn($sub) => strcmp($sub['category'], 'Toys and Games') === 0);
+                    @endphp
+                    @foreach ($subcat as $sub)
+                    @php
+                    $subURL = "https://".$_SERVER['SERVER_NAME']."/products/Toys%20and%20Games?subcategory=".str_replace(" ","%20",$sub['subcategory']);
+                    @endphp
+                    <a class="category-item-link" href="<?php echo e($subURL) ?>">{{$sub['subcategory']}}</a>
+                    @endforeach
+                  </div>
+                </div>
+              </div>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Contact</a>
+            </li>
+            <li class="nav-item small-screen">
+              @if(Illuminate\Support\Facades\Auth::check())
+              <a class="nav-link" href="{{ route('logout') }}">Logout</a>
+              @else
+              <a class="nav-link" href="{{ route('login') }}">Login</a>
+              @endif
+            </li>
+            @if(Illuminate\Support\Facades\Auth::check())
+            <li class="nav-item small-screen">
+              <a class="nav-link" href="{{ route('cart') }}">Shopping Cart</a>
```

Comment:
As the cart route doesn't exist, this causes unhandled exceptions to be thrown as soon as a user logs in. Can you create a placeholder cart route or simply remove this link for the moment?